src/drivers/pwm_out_sim/pwm_out_sim.cpp

```diff
@@ -547,6 +575,15 @@ PWMSim::task_main()
 			_armed = aa.armed;
 			_failsafe = aa.force_failsafe;
 			_lockdown = aa.manual_lockdown;
+
+			param_t param_handle;
+			
+			// maximum motor slew rate parameter
+			param_handle = param_find("MOT_SLEW_MAX");
```

Comment:
Should I change this in https://github.com/PX4/Firmware/blob/master/src/modules/sensors/sensor_params.c#L3448